pyvcloud/vcd/org.py

```diff
@@ -118,6 +118,27 @@ def get_catalog(self, name):
                 return self.client.get_resource(link.href)
         raise Exception('Catalog not found.')
 
+    def get_user(self, user_name):
```

Comment:
Please add a doc string to every new functionality method you introduce in pyvcloud.